docs/sources/enterprise/saml.md

```diff
@@ -204,6 +204,61 @@ org_mapping = Engineering:2, Sales:3
 allowed_organizations = Engineering, Sales
 ```
 
+## Set up SAML with Okta
+
+To configure SAML integration with Okta, create integration inside the Okta org first.
+
+1. Log in to the [Okta portal](https://login.okta.com/).
+1. Go to the Admin Console in your Okta org by clicking **Admin** in the upper-right corner. If you are in the Developer Console, click **Developer Console** in the upper-left corner and then click **Classic UI** to switch over to the Admin Console.
```

Comment:
```suggestion
1. Go to the Admin Console in your Okta organization by clicking **Admin** in the upper-right corner. If you are in the Developer Console, then click **Developer Console** in the upper-left corner and then click **Classic UI** to switch over to the Admin Console.
```

Reply:
If Okta uses "org" in their UI, then leave that word as-is. When possible, write out the word or match the UI.

Reply:
Okta uses `org` in the docs (this guide based on Okta SAML configuration docs), but in the UI I found `organization`, not `org`, so I think your suggestion is better than original docs.